example/src/pages/App.tsx

```diff
@@ -209,23 +207,47 @@ class App extends React.Component<any, any> {
       this.setState({ wallyClientId: provider.clientId });
 
       await this.subscribeProvider(provider);
-      await provider.requestAccounts();
 
       const web3: any = initWeb3(provider);
-      const accounts = await web3.eth._provider.selectedAddress;
+      await web3.eth.requestAccounts();
+
+      const accounts = await web3.eth.currentProvider.selectedAddress;
       const address = accounts;
 
-      // Needs to be included in the SDK to get the networkId and chainId
-      const networkId = 1;
-      const chainId = 1;
+      const bearerToken = process.env.REACT_APP_WALLY_TOKEN;
+      const host = process.env.REACT_APP_WALLY_HOST;
+      const resp = await fetch(`${host}/app`, {
+        method: "GET",
+        headers: {
+          Authorization: `Bearer ${bearerToken}`
+        }
+      });
+
+      const data = await resp.text();
+      console.log(JSON.parse(data).environment);
+
+      const wallyEnvironment = JSON.parse(data).environment;
+
+      const networkTypes = [
+        { id: 1, name: "mainnet", wallyEnvironment: "ETHEREUM" },
+        { id: 5, name: "goerli", wallyEnvironment: "ETHEREUM_GOERLI" }
+      ];
+
+      const networkId = networkTypes.find(
+        network => network.wallyEnvironment === wallyEnvironment
+      )?.id;
+
+      const chainId = networkTypes.find(
+        network => network.wallyEnvironment === wallyEnvironment
+      )?.id;
```

Comment:
do the `web3` methods for `networkId` and `chainId` still not work? should be fixed in later versions of the SDK - lmk if they're still missing